src/paymasterclient.ts

```diff
@@ -48,36 +57,99 @@ export type Bundle = {
   readonly ChainID: number
 }
 
-export class PaymasterClient extends ethers.JsonRpcProvider {
-  constructor(url?: string | FetchRequest, network?: Networkish, options?: JsonRpcApiProviderOptions) {
-    super(url, network, options)
+export class PaymasterClient {
+  private sponsorClient: ethers.JsonRpcProvider
+  private userClient: ethers.JsonRpcProvider
+
+  constructor(
+    userUrl: string | FetchRequest,
+    sponsorUrl: string | FetchRequest,
```

Comment:
what if a user:
1. just want to use the user api?
2. just want to use the sponsor api and private policy?
3. will use both of them.
But now, I have to input two URL.  I think we can improve  so that user just need input necessary one.

Reply:
Make the sponsorUrl optional, so that private policy UUID can only be sent if the user provides this URL. Add a comment explaining the purpose of sponsorUrl and Options.